src/main/java/ru/vyarus/spock/jupiter/engine/context/ClassContext.java

```diff
@@ -17,8 +18,13 @@
  */
 public class ClassContext extends AbstractContext {
 
+    @SuppressWarnings("unused")
     public ClassContext(final ExtensionRegistry registry, final SpecInfo spec) {
```

Comment:
Why second constructor? It used just once (and it is not a public api - no need to keep compatibility)

Reply:
Didn't want to have any breaking changes, but since you've confirmed it's ok, I've removed it.